### PR TITLE
[Shared Lending Platform] update to newest design / copy

### PR DESCRIPTION
**Pull request by billhimmelsbach**
updates the copy on the following pages to align with figma designs and discussion:

- unauthenticated landing page
- complete user profile form
- warning message for personal email
- authenticated landing form
- view user profile
- view institution details

### CODE REVIEW

File: src/pages/ProfileForm/Step1Form/Step1FormHeader.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ function Step1FormHeader(): JSX.Element {
       <TextIntroduction
         className='mb-[3.75rem]'
         heading='Complete your user profile'
-        subheading='Please complete the fields below and select the financial institution you are authorized to file for. Once you have successfully associated your user profile with a financial institution you will have access to the filing platform and can begin the filing process.'
+        subheading='Complete the fields below and select the financial institution you are authorized to file for. Once you have successfully associated your user profile with a financial institution you will have access to the filing platform and can begin the filing process.'
```
Comment (natalia-fitzgerald) [inline src/pages/ProfileForm/Step1Form/Step1FormHeader.tsx:15]: Change "filing platform" to "platform"

Comment (billhimmelsbach) [reply src/pages/ProfileForm/Step1Form/Step1FormHeader.tsx]: Updated!

https://github.com/cfpb/sbl-frontend/pull/206/commits/a9da76dc05ca1a76e0e12a2bd65ed97444a16d12

**Review by natalia-fitzgerald — APPROVED** [review]
@billhimmelsbach 
Thanks for your work on this. looks great!

**Comment by billhimmelsbach** [comment]
Great! 🎉

**Pull request merged**